src/Components/Endpoints/src/Results/RazorComponentResultExecutor.cs

```diff
@@ -30,11 +35,67 @@ public virtual Task ExecuteAsync(HttpContext httpContext, RazorComponentResult r
             response.StatusCode = result.StatusCode.Value;
         }
         
-        return RazorComponentEndpoint.RenderComponentToResponse(
+        return RenderComponentToResponse(
             httpContext,
             result.RenderMode,
             result.ComponentType,
             result.Parameters,
             result.PreventStreamingRendering);
     }
+
+    internal static Task RenderComponentToResponse(
```

Comment:
Would there be any merit to reusing some of the logic between this method and `RazorComponentEndpointInvoker.RenderComponentCore()`, or will they diverge over time?

Reply:
In addition to that, the other pipeline will handle form posts, which this will definitely not do.